### ISSUE TITLE
Add documentation to README about the new .ipfsignore file

### ISSUE THREAD

**Issue by sohkai** [aragon/aragon-cli#135]

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 0.001 ETH (0.23 USD @ $232.7/ETH)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/aragon/aragon-cli/135/1266).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/aragon/aragon-cli/135/1266).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $22,838.41 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 0.001 ETH (0.23 USD @ $232.7/ETH)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/aragon/aragon-cli/135/1266).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/aragon/aragon-cli/135/1266).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $22,838.41 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
💰 A crowdfund contribution worth 0.00010  ETH (0.02 USD @ $232.7/ETH) has been attached to this funded issue  from @.💰 

Want to chip in also? Add your own contribution [here](https://gitcoin.co/issue/aragon/aragon-cli/135/1266).

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 12 months from now.
Please review their action plans below:


**1) [aj07](https://gitcoin.co/profile/aj07) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?mutate_worker_action=approve&worker=aj07) | [reject worker](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?mutate_worker_action=reject&worker=aj07))_.**

I would like to work on this. I have a querry.

How big you want documentation to be ?
Is there any specific point that should be highlighted of IPFS?

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/aragon/aragon-cli/135/1266).

**Comment by aj07** [comment]
@gitcoinbot just wanted to confirm, is this issue of 0.02$?

**Comment by Quazia** [comment]
No, I was just playing around with the gitcoin system since I knew you'd be taking this issue. We'll use the Giveth DApp and can discuss how long the issue takes you and come to something we both think is fair!

**Comment by gitcoinbot** [comment]
@aj07 Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@aj07 Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

@aj07 due to inactivity, we have escalated [this issue](https://gitcoin.co/issue/aragon/aragon-cli/135/1266) to Gitcoin's moderation team. Let us know if you believe this has been done in error!

* [x] warning (3 days)
* [x] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

@aj07 due to inactivity, we have escalated [this issue](https://gitcoin.co/issue/aragon/aragon-cli/135/1266) to Gitcoin's moderation team. Let us know if you believe this has been done in error!

* [x] warning (3 days)
* [x] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by aj07** [comment]
https://github.com/aj07/aragon-cli/pull/1

**Comment by aj07** [comment]
@Quazia or @sohkai Will you Please close this, If things are ok

**Comment by sohkai** [comment]
@aj07 	could you make the pull request against this repo? I think you've made it against your own fork :).

**Comment by aj07** [comment]
ahh.. .got it. Done @sohkai

**Comment by frankchen07** [comment]
It looks like this issue is closed out. @aj07 did you get paid out? If yes, we can mark the bounty as done.

**Comment by aj07** [comment]
@frankchen07, No Gitcoin has limitation of working 3 issue at a time. I wanted to work on another issue. So, I withdrew from bounty. But, anyway please merge if patch works fine to your requirement.

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by ceresstation** [comment]
Hey @aj07 are you able to apply + submit work once one of those bounties is complete so that @sohkai can pay you out?

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by ryan-shea** [comment]
Hey @saurav-malani, any updates here? Just checking in :smil

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by gitcoinbot** [comment]
@saurav-malani Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=1) | [3 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=3) | [5 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=5) | [10 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=10) | [100 days](https://gitcoin.co/issue/aragon/aragon-cli/135/1266?snooze=100)

**Comment by ryan-shea** [comment]
@saurav-malani any updates here?

**Comment by sohkai** [comment]
@ryan-shea This bounty should be closed or award to @0xGabi instead (see 65da26f).
